runner_manager/models/backend.py

```diff
@@ -46,3 +47,36 @@ class DockerConfig(BackendConfig):
     """Configuration for Docker backend."""
 
     base_url: str = "unix:///var/run/docker.sock"
+
+
+class GCPConfig(BackendConfig):
+    """Configuration for GCP backend."""
+
+    project_id: str
+    zone: str
+    service_account_email: str = "default"
+
+
+class GCPInstanceConfig(InstanceConfig):
+    image_family: str
+    image_project: str
+    machine_type: str
```

Comment:
I wouldn't mind having some default values:
```suggestion
    image_family: str = "ubuntu-2004-lts"
    image_project: str = "ubuntu-os-cloud"
    machine_type: str = "e2-small"
```

Reply:
ok